### ISSUE TITLE
Custom Protocol issue for Windows + History API

### ISSUE THREAD

**Issue by andreszorro** [tauri-apps/tauri#1642]
Hello team! Not sure if this is a bug or something I'm missing. I just migrated from Tauri 0.14.1 to 1.0.0-beta-rc.4 and I noticed a discrepancy in the builds between Windows and MacOS. I really hope this is only a misconfiguration from my end.

**To Reproduce**
1. Compile in Mac and Windows any application that uses the Browser History API

**Expected behavior**
Both compilations work with Browser History API.

**Screenshots**
In MacOS
![Screen Shot 2021-04-28 at 1 48 31 PM](https://user-images.githubusercontent.com/6423334/116457859-8d23ef00-a829-11eb-9db9-b438bce6b16a.png)

In Windows
![Screen Shot 2021-04-28 at 1 48 20 PM](https://user-images.githubusercontent.com/6423334/116457898-98771a80-a829-11eb-84a7-9d353ad87d13.png)

**Platform and Versions (please complete the following information):**
OS: Windows
Node.js environment
  Node.js - 14.16.1
  @tauri-apps/cli - 1.0.0-beta-rc.4
  @tauri-apps/api - 1.0.0-beta-rc.3

Global packages
  npm - 6.14.12
  yarn - 1.22.10

Rust environment
  rustc - 1.51.0
  cargo - 1.51.0

Cargo.toml
```
[package]
name = "myapp"
version = "0.1.0"
description = "A Tauri App"
authors = [ "you" ]
license = ""
repository = ""
default-run = "myapp"
edition = "2018"
build = "src/build.rs"

[build-dependencies]
tauri-build = { version = "1.0.0-beta-rc.0" }

[dependencies]
serde_json = "1.0"
serde = { version = "1.0", features = [ "derive" ] }
tauri = { version = "1.0.0-beta-rc.4", features =["api-all"]}

[target."cfg(windows)".build-dependencies]
winres = "0.1"

[features]
custom-protocol = [ "tauri/custom-protocol" ]
default = [ "custom-protocol" ]

[[bin]]
name = "clearview-sonarqube-fetcher"
path = "src/main.rs"
```

tauri.conf.json
```
{
  "build": {
    "distDir": "../build",
    "devPath": "http://localhost:3000",
    "beforeDevCommand": "",
    "beforeBuildCommand": ""
  },
  "tauri": {
    "bundle": {
      "active": true,
      "targets": "all",
      "identifier": "com.tauri.dev",
      "icon": [
        "icons/32x32.png",
        "icons/128x128.png",
        "icons/128x128@2x.png",
        "icons/icon.icns",
        "icons/icon.ico"
      ],
      "resources": [],
      "externalBin": [],
      "copyright": "",
      "category": "DeveloperTool",
      "shortDescription": "",
      "longDescription": "",
      "deb": {
        "depends": [],
        "useBootstrapper": false
      },
      "macOS": {
        "frameworks": [],
        "minimumSystemVersion": "",
        "useBootstrapper": false,
        "exceptionDomain": "",
        "signingIdentity": null,
        "entitlements": null
      },
      "windows": {
        "certificateThumbprint": null,
        "digestAlgorithm": "sha256",
        "timestampUrl": ""
      }
    },
    "updater": {
      "active": false
    },
    "allowlist": {
      "all": true
    },
    "windows": [
      {
        "title": "ClearView: SonarQube Fetcher",
        "width": 500,
        "height": 600,
        "resizable": false,
        "fullscreen": false
      }
    ],
    "security": {
      "csp": "default-src blob: data: filesystem: ws: http: https: 'unsafe-eval' 'unsafe-inline'"
    }
  }
}
```

**Comment by lucasfernog** [comment]
Did we fix this on wry already @amrbashir ?

**Comment by andreszorro** [comment]
@lucasfernog Found the PR in [wry repo](https://github.com/tauri-apps/wry/pull/173), and indeed seems like the workaround was merged for wry#0.9.0. Running `cargo tree`, found out that the version 1.0.0-beta-rc4 is referencing wry#0.8.0. So I guess it's just a matter of updating the dependency.

I'm kind of a newbie in Rust. Is there a way to override the wry dependency locally while you guys publish it?

**Comment by lucasfernog** [comment]
I have an opened PR that updates wry and I'll merge it later tonight. You can pull the tauri crate from git on Cargo.toml to use it. I'll close this issue, please reopen if something is buggy.
